packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -471,31 +537,32 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   }
 }
 
-.mdc-textfield--multiline {
-  $padding-inset: 4px;
+.mdc-textfield--textarea {
+  $padding-inset: 16px;
   $label-offset-y: $padding-inset + 2;
   $label-offset-x: $padding-inset;
 
   display: flex;
   height: initial;
   transition: none;
+  border: 1px solid $mdc-textarea-border-on-light;
+  border-radius: 4px;
+
+  @include mdc-theme-dark(".mdc-textfield") {
+    border-color: $mdc-textarea-border-on-dark;
+  }
 
   &::after {
```

Comment:
I suspect this can be removed, since content is no longer set now that bottom-line has its own element.